### ISSUE TITLE
should accept kwargs parameter in loaders.from_file also

### ISSUE THREAD

**Issue by arvindjangra1996** [kiwicom/schemathesis#496]
I am using runner for my apis testing. I am passing json string in  `runner.prepare` as schema_uri parameter
```
# in loaders.py
def from_file(
    file: Union[IO[str], str],
    location: Optional[str] = None,
    base_url: Optional[str] = None,
    method: Optional[Filter] = None,
    endpoint: Optional[Filter] = None,
    tag: Optional[Filter] = None,
    *,
    app: Any = None,
    validate_schema: bool = True,
    **kwargs:Any,    <----------------------------------
) -> BaseSchema:
```

**Comment by Stranger6667** [comment]
Hi @arvindjangra1996 
How `kwargs` should be used in `from_file`? Could you, please provide some example use case?

**Comment by Stranger6667** [comment]
Ah, got it, I'll get back to you soon

**Comment by arvindjangra1996** [comment]
hi.. i am trying to pass my json string with loader = loaders.from_file but it is not accepting additional params like headers. I think code is written only for loaders.from_uri
```
runner.prepare(
        schema_uri=my_json_string,
        loader=loaders.from_file,
        headers=api_headers)
```

**Comment by Stranger6667** [comment]
Thank you for examples :) I am working on fixing this

**Comment by arvindjangra1996** [comment]
thanks :)

**Comment by Stranger6667** [comment]
The fix is released in `1.1.2` (https://pypi.org/project/schemathesis/1.1.2/)
I am going to add extra input validation in `runner.prepare` in the next one.

Let me know if the fix works for you

**Comment by arvindjangra1996** [comment]
yeah its working now. Thanks :).
